### DISCUSSION TITLE
Opis efektu pomijania karty

### DISCUSSION THREAD

**Discussion by itchystone** [emkarcinos/WMIAdventure#655] [Codding]
![image](https://user-images.githubusercontent.com/43492764/141270224-b6e16be1-d8fc-4125-8ed2-e3c5f8d08b3b.png)

Może opis zmienić na "pomija następną kartę przeciwnika"?

**Comment by emkarcinos** [comment]
Nie pomija karty, tylko pomija efekty, chyba chodzi o to że pominięcie karty sugeruje że się wykona następna

**Reply by itchystone** [reply]
ale tak jest przecież, skipcard na backendzie sprawia, że karta jest pomijana (sciągana z góry i wstawiana na tył) iwykonuje się następna

**Reply by itchystone** [reply]
blokowanie karty blokuje efekty

**Reply by mattyl006** [reply]
to dobrze że tak jest, jak wymyślałem, to o to mi chodziło
